static/app/views/issueList/pages/dynamicGrouping.tsx

```diff
@@ -1023,50 +1052,23 @@ const ClusterTitle = styled('h3')`
   word-break: break-word;
 `;
 
-// Zone 2: Stats section with visual hierarchy
-const StatsSection = styled('div')`
-  padding: ${space(2)} ${space(3)};
-  background: ${p => p.theme.backgroundSecondary};
-  border-top: 1px solid ${p => p.theme.innerBorder};
-  border-bottom: 1px solid ${p => p.theme.innerBorder};
+// Horizontal stats bar below header
+const ClusterStatsBar = styled('div')`
   display: flex;
-  justify-content: space-between;
-  align-items: center;
-  gap: ${space(2)};
   flex-wrap: wrap;
-`;
-
-const PrimaryStats = styled('div')`
-  display: flex;
   align-items: center;
-  gap: ${space(3)};
+  gap: ${space(2)};
```

Comment:
especially since this is a new page - any reason we need to keep using the deprecated `space` instead of `theme.space`

Reply:
none of this is going to live very long